src/modules/admin/Admin.scss

```diff
@@ -0,0 +1,4 @@
+.adminContainer {
```

Comment:
Use spacing from Templates.scss - file margins padding and sizes should be same

`  @include col;
  padding: 60px;

  & > form {
    margin: 62px auto 82px auto;
  }`